admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -289,176 +305,178 @@ var formCategory = (function() {
       data: {
         'form[category][name]': $('#form_step1_new_category_name').val(),
         'form[category][id_parent]': $('#form_step1_new_category_id_parent').val(),
-        'form[_token]': $('#form #form__token').val()
+        'form[_token]': $('#form #form__token').val(),
       },
-      beforeSend: function() {
+      beforeSend() {
         $('button.submit', elem).attr('disabled', 'disabled');
         $('ul.text-danger', elem).remove();
         $('*.has-danger', elem).removeClass('has-danger');
         $('*.has-danger').removeClass('has-danger');
       },
-      success: function(response) {
-        //inject new category into category tree
-        var html = '<li>' +
-          '<div class="checkbox js-checkbox">' +
-            '<label>' +
-              '<input type="checkbox" name="form[step1][categories][tree][]" checked value="'+response.category.id+'"> ' +
-                response.category.name[1] +
-                '<input type="radio" value="'+response.category.id+'" name="ignore" class="default-category">' +
-            '</label>' +
-          '</div>' +
-          '</li>';
-
-        var parentElement = $('#form_step1_categories input[value='+response.category.id_parent+']').parent().parent();
-        if(parentElement.next('ul').length === 0){
-          html = '<ul>' + html + '</ul>';
+      success(response) {
+        // inject new category into category tree
+        let html = `${'<li>'
```

Comment:
```suggestion
        let html = `${'<li>'
```

Where is the end of `${` ?